package.json

```diff
@@ -39,8 +39,8 @@
     ]
   },
   "devDependencies": {
```

Comment:
- [ ] It would be better to improve our README file in order for the README to be descriptive. Here is what makes it descriptive; 
  - [ ]  Title of the project, In our case, we can be a little specific.
  - [ ] Description about it
It would be really helpful if we use the [Microverse](https://github.com/microverseinc/readme-template) recommended template

Reply:
Thanks for the heads up. Done